src/main/java/com/dev/cinema/validation/EmailValidation.java

```diff
@@ -0,0 +1,19 @@
+package com.dev.cinema.validation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Documented
+@Constraint(validatedBy = EmailValidator.class)
+@Target({ElementType.METHOD, ElementType.FIELD })
```

Comment:
do we need to put this annotation on method ?

Reply:
done